openshift-tests-plugin/plugin/platform.sh

```diff
@@ -44,6 +44,25 @@ EOF
     echo "${OPENSHIFT_TESTS_EXTRA_ARGS}" > /tmp/shared/platform-args
 }
 
+
+
+function setup_provider_gce() {
```

Comment:
We standardize the function name with the platform name from the [Infrastructure Object API][API], in this case it is GCP
```suggestion
function setup_provider_gcp() {
```

Additionally you might need to add a call to this function when the platform is discovered[1] as GCP here: https://github.com/redhat-openshift-ecosystem/provider-certification-plugins/blob/0732fc4280c3d9388f563814c24bb9939f61274c/openshift-tests-plugin/plugin/platform.sh#L120-L125

[1] platform type discovery:
https://github.com/redhat-openshift-ecosystem/provider-certification-plugins/blob/0732fc4280c3d9388f563814c24bb9939f61274c/openshift-tests-plugin/plugin/platform.sh#L116-L118

[API]: https://docs.openshift.com/container-platform/4.17/rest_api/config_apis/infrastructure-config-openshift-io-v1.html#status-platformstatus-gcp